sofascoreAcademy/sofascoreAcademy/SettingsViewController.swift

```diff
@@ -41,9 +85,49 @@ class SettingsViewController: UIViewController, BaseViewProtocol {
         title = "Settings"
         dismissButton.setTitle("Dismiss", for: .normal)
         dismissButton.addTarget(self, action: #selector(dismissSettings), for: .touchUpInside)
+        
+        nameLabel.text = UserDefaults.standard.string(forKey: "name")
+        nameLabel.backgroundColor = .clear
+        nameLabel.font = .headlineBold32
+        nameLabel.textColor = .primaryBlack
+        nameLabel.numberOfLines = 2
+        
+        eventCountLabel.backgroundColor = .clear
+        eventCountLabel.font = .regular14
+        eventCountLabel.textColor = .secondaryGray
+        
+        leagueCountLabel.backgroundColor = .clear
+        leagueCountLabel.font = .regular14
+        leagueCountLabel.textColor = .secondaryGray
+        
+        logoutButton.setTitle("Logout", for: .normal)
+        logoutButton.backgroundColor = .headerBackground
+        logoutButton.layer.cornerRadius = 10
+        logoutButton.addTarget(self, action: #selector(logoutButtonTapped), for: .touchUpInside)
     }
     
     @objc func dismissSettings() {
         navigationController?.popViewController(animated: true)
     }
+    
+    @objc func logoutButtonTapped() {
+        UserDefaults.standard.removeObject(forKey: "name")
+        
+        let keychain = Keychain(service: "com.academy")
+        try? keychain.remove("token")
+        
+        clearDB()
+        
+        let loginVC = LoginViewController()
+        let navController = UINavigationController(rootViewController: loginVC)
+        navController.modalPresentationStyle = .fullScreen
+        present(navController, animated: true)
```

Comment:
Želimo izbrisati hierarhiju, ovdje treba na direktno na window postaviti novi UINavigationController(rootViewController: loginVC).
Možeš neki manager imati za to.